src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -412,26 +426,25 @@ internal ImmutableArray<IVariableDeclaratorOperation> CreateVariableDeclarator(B
             }
         }
 
-#nullable enable
         private IPlaceholderOperation CreateBoundDeconstructValuePlaceholderOperation(BoundDeconstructValuePlaceholder boundDeconstructValuePlaceholder)
         {
             SyntaxNode syntax = boundDeconstructValuePlaceholder.Syntax;
             ITypeSymbol? type = boundDeconstructValuePlaceholder.GetPublicTypeSymbol();
             bool isImplicit = boundDeconstructValuePlaceholder.WasCompilerGenerated;
             return new PlaceholderOperation(PlaceholderKind.Unspecified, _semanticModel, syntax, type, isImplicit);
         }
-#nullable disable
 
         private IDeconstructionAssignmentOperation CreateBoundDeconstructionAssignmentOperator(BoundDeconstructionAssignmentOperator boundDeconstructionAssignmentOperator)
         {
+            IOperation target = Create(boundDeconstructionAssignmentOperator.Left);
+            // Skip the synthetic deconstruction conversion wrapping the right operand.
```

Comment:
This describes what we are doing, can you describe a bit about "why" we are doing it?